openVpnConfig/ap-northeast-1.conf

```diff
@@ -0,0 +1,181 @@
+client
+dev tun
+proto udp
+remote ap-northeast-1.vpn.1nce.com 1194
+resolv-retry infinite
+nobind
+explicit-exit-notify 3
+keepalive 5 30
+user root
```

Comment:
I don't see the `user` and `group` in the downloadable config (from 1NCE portal) and in other openVpnConfig files.

What is the actual purpose of adding it ?